CONTRIBUTING.md

```diff
@@ -45,6 +45,6 @@ Besides publishing a new version to npm, semantic-release also creates a git tag
 on GitHub, generates changelogs from the commit messages and puts them into the release notes.
 
 Before the publish it runs the `npm run build` script which generates type definitions for Flow & Typescript based on the [templates](scripts/templates/).
-The script also generates the API docs. After the publish, the API docs are automatically pushed to the `gh-pages` branch which updates the documentation at  [octokit.github.io/node-github](http://octokit.github.io/node-github/).
+The script also generates the API docs. After the publish, the API docs are automatically pushed to the `gh-pages` branch which updates the documentation at  [octokit.github.io/rest.js](http://octokit.github.io/rest.js).
```

Comment:
We should create a redirect in https://github.com/octokit/octokit.github.com similar to what we did when we first moved node-github to this org.

Reply:
I got it in the todos, I moved them from the issue to the PR so they will be harder to miss

Reply:
Recommend using the URL with `https`

Reply:
fixed it, cheers!